{
 "0.3.2": {
  "meta": {
   "description": "Use Rust failures as Python exceptions",
   "homepage": "https://github.com/untitaker/shippai",
   "license": {
    "fullName": "MIT License",
    "shortName": "mit",
    "spdxId": "MIT",
    "url": "https://spdx.org/licenses/MIT.html"
   },
   "long_description": "",
   "maintainers": [
    {
     "email": "gebner@gebner.org",
     "github": "gebner",
     "githubId": 313929,
     "name": "Gabriel Ebner"
    }
   ],
   "name": "python3.8-shippai",
   "platforms": [
    "aarch64-linux",
    "armv5tel-linux",
    "armv6l-linux",
    "armv7a-linux",
    "armv7l-linux",
    "mipsel-linux",
    "i686-linux",
    "x86_64-linux",
    "powerpc64le-linux",
    "riscv32-linux",
    "riscv64-linux",
    "x86_64-darwin",
    "i686-darwin",
    "aarch64-darwin",
    "armv7a-darwin"
   ],
   "position": "pkgs/development/python-modules/shippai/default.nix:14"
  },
  "revs": [
   "00022fbeda385d7b6ae2eee44f07eecfc6d92015",
   "5101b357ec30c1ac4a9273b379e631f841af9fa3"
  ]
 }
}